engines/metaengine.h

```diff
@@ -116,6 +116,20 @@ class MetaEngine : public PluginObject {
 	}
 
 	/**
+	* Return whether engine's saves could be detected with
+	* "<target>.###" pattern and "###" corresponds to slot
+	* number.
+	*
+	* If that's not true or engine is using some unusual way
+	* of detecting saves and slot numbers, this should return
+	* false. In that case Save/Load dialog would be unavailable
+	* during cloud saves sync.
+	*
+	* @return	true, if "<target>.###" is OK for this engine
+	*/
+	virtual bool simpleSaveNames() const { return false; }
```

Comment:
Wouldn't the `simpleSaveNames` thing fit better as an engine feature? (See `MetaEngineFeature`)